src/components/Chat/ChatInput/SendMessageButton.tsx

```diff
@@ -45,56 +38,39 @@ const SendIconTooltip = ({
   );
 };
 
-export const SendMessageButton = ({ handleSend, isInputEmpty }: Props) => {
-  const { t } = useTranslation('chat');
+interface Props {
+  handleSend: () => void;
+  isDisabled: boolean;
+  disabledTooltip?: string;
+}
+
+export const SendMessageButton = ({
+  handleSend,
+  isDisabled,
+  disabledTooltip,
```

Comment:
we display tooltip even when not disabled. I would rename to `tooltip`

Reply:
fixed